### ISSUE TITLE
Does not reduced CrammingBERT training time 

### ISSUE THREAD

**Issue by tbaggu** [Liuhong99/Sophia#18]
Hi 

I have integrated Cramming-BERT code with this optimizer but i have not seen any training time reduction rather it is increased by one hour/

https://github.com/JonasGeiping/cramming/tree/main/cramming

**Comment by Liuhong99** [comment]
Could you please elaborate on how you integrated Cramming-BERT code with Sophia? Especially, learning rate, beta1, beta2, weight decay for Adam and Sophia you used in Cramming-BERT. In the early stage of Sophia, I tried BERT-base on the academic budget Bert code base (early version of Sophia) and was able to achieve considerable speedup. More context on your issues will be helpful.

**Comment by tbaggu** [comment]
@Liuhong99  Here are the details

I have added above below into the repo mentioned 

```
cramming/config/train/optim/SophiaG.yaml

type: SophiaG

lr: 2e-4
betas:
  - 0.965
  - 0.99
rho: 0.01
weight_decay: 1e-1

cramming/config/train/bert-o3.yaml

defaults:
  - optim: SophiaG
  - optim_mod: disabled

cramming/backend/optimizers/Sophia.py

cramming/backend/torch_default.py

from .optimizers.Sophia import SophiaG

    elif cfg_train.optim.type == "SophiaG":
        optimizer_class = SophiaG

```
for adam i have used below hyper params

```
type: AdamW

lr: 0.0005
betas:
  - 0.9
  - 0.98
eps: 1e-6
weight_decay: 0.01
amsgrad: False

```

**Comment by Liuhong99** [comment]
seems like you are using quite conservative lr rho and wd for Sophia. Maybe you can try lr=4e-4, rho=0.02, and wd=0.015
